config/routes.rb

```diff
@@ -9,13 +9,11 @@
     get 'login', to: 'sessions#new'
     post 'login', to: 'sessions#create'
 
+    get 'logout', to: 'sessions#destroy'
```

Comment:
Do you need both of these ?

Reply:
Both of which? The routes here are all for different controller actions/request methods so need different routes, so I'm not sure I completely understand what you're asking . PLssss clarify if you still want an answer on this